guides/posts/pratham/create-a-text-sentiment-analysis-app/post.md

```diff
@@ -0,0 +1,318 @@
+    alt="react app folder structure"
+  />
+</figure>
+
+**2. Create a Form**
+
+We want a form in our application where users can add the test in order to retrieve the sentiment. We just need an input field and a submit button in the form.   
+
+Go to your project folder (sentiment-analysis that we just created in the first step) and you’ll find that React created a bunch of other folders and files for us. We don’t need to check them all right now.   
+
+Just Go to the `sentiment-analysis folder > src > App.js` file and create a form with an URL input field and a submit button.
+
+```js
+function App() {
+  return (
+    <div>
+      <form action=””>
+        <input type=”text” name=”text” id=”text” placeholder=”Enter your text here.” />
+        <input className=”btn” type=”submit” value=”Analyze” />
+      </form>
+    </div>
+  );
+}
+export default App;
+```
+
+Run the application to check the output by running the following command from the root of the project and go to https://localhost:3000
+
+`npm start`
+
+<figure>
+  <img
+    src="https://raw.githubusercontent.com/PrathamKumar14/DevRel-Stack-Data/dev/guides/posts/pratham/create-a-text-sentiment-analysis-app/app.png"
+    alt="text sentiment analysis application"
+  />
+</figure>
+
+**3. Copy-paste the code snippet from RapidAPI Hub**
+
+We are all set to integrate Text Analysis API code into our application. You don’t even need to write the code. RapidAPI provides the code snippet in 20 programming languages with 40 libraries’ support.   
+
+Go to the [Test Analysis](https://rapidapi.com/gaurmanojkumar530/api/text-analysis12/?utm_source=medium.com%2F%40Rapid_API&utm_medium=DevRel&utm_campaign=DevRel) API homepage and copy the code snippet from the Copy Snippets section of the API playground.   
+
+Select the JavaScript language and [axios](https://github.com/axios/axios) method from the dropdown menu and click on the “Copy Code” button.
+
+<figure>
+  <img
+    src="https://raw.githubusercontent.com/PrathamKumar14/DevRel-Stack-Data/dev/guides/posts/pratham/create-a-text-sentiment-analysis-app/text-analysis-copy-code.png"
+    alt="copy code snippet of text analysis API"
+  />
+</figure>  
+
+Create a function `fetchData()` in the App.js file and paste the code inside that function.  
+
+We need to install axios library first in order to use it in our application. Run the following command from the root of the project.
+
+`npm install axios`
+
+**4. Let the user enter the text and get the sentiment** 
+
+We are almost done! We just need to validate the form so that the user can enter the text to get the result.  
+
+Let’s complete the form first. Create two `useState` hooks in the App.js file, one for handling the state of text that the user enters and the second for loading while API fetching the response from the server. 
+ 
+```js
+const [text, setText] = useState(“”);
```

Comment:
I think that these quotes will throw errors if copied and pasted. You need to use `""` instead of `“”`.